src/main/java/com/pjait/byt/pettracker/Collar.java

```diff
@@ -19,8 +20,13 @@
     }
 
     public Collar(int size, String collarColour) {
-        this.size = size;
```

Comment:
for this @Size(min=1, max=4) would have been a better solution but this also works (from the validation API)